par_shapes.h

```diff
@@ -807,10 +820,19 @@ void par_shapes_scale(par_shapes_mesh* m, float x, float y, float z)
         *points++ *= z;
     }
     float* n = m->normals;
-    if (n && (x != y || x != z || y != z)) {
-        x = 1.0f / x;
-        y = 1.0f / y;
-        z = 1.0f / z;
+    if (n && !(x == y && y == z)) {
```

Comment:
This condition was simplified: only modify normals if the scaling factors are not identical. If `x==y` and `y==z` then we know `x == z`